src/main/java/com/algorand/algosdk/v2/client/indexer/LookupAssetByID.java

```diff
@@ -16,6 +16,9 @@
 
     private Long assetId;
 
+    /**
+     * @param assetId
```

Comment:
Is there documentation we could add to the spec for these?

Reply:
Couldn't find it. 
What we can do is skip the comments for those.

Reply:
I think you can keep it, it's a good idea. The spec is probably just missing descriptions for the path parameters. These are sort of internal methods anyway.